server/http.go

```diff
@@ -103,6 +103,7 @@ func (s *HTTPd) RegisterSiteHandlers(site site.API, cache *util.Cache) {
 		"buffersoc":     {[]string{"POST", "OPTIONS"}, "/buffersoc/{value:[0-9.]+}", floatHandler(site.SetBufferSoc, site.GetBufferSoc)},
 		"prioritysoc":   {[]string{"POST", "OPTIONS"}, "/prioritysoc/{value:[0-9.]+}", floatHandler(site.SetPrioritySoc, site.GetPrioritySoc)},
 		"residualpower": {[]string{"POST", "OPTIONS"}, "/residualpower/{value:[-0-9.]+}", floatHandler(site.SetResidualPower, site.GetResidualPower)},
+		"autocharge":    {[]string{"POST", "OPTIONS"}, "/autochargecostlimit/{value:[-0-9.]+}", floatHandler(site.SetAutoChargeCostLimit, site.GetAutoChargeCostLimit)},
```

Comment:
Der Standard-Wert wäre 0, heißt ohne etwas konfiguriert zu haben würden wir bei negativen Preisen immer laden?
Alternativ könnten wir noch einen `DELETE` Endpoint einbauen (analog targettime) um das limit bewusst deaktivieren.

Reply:
> Der Standard-Wert wäre 0, heißt ohne etwas konfiguriert zu haben würden wir bei negativen Preisen immer laden?

Was ja durchaus charmant wäre? Alternativ können wir im Code auch prüfe ob der Wert != 0 ist um ihn als aktiv zu betrachten. Dann brauchts kein API.

Reply:
Mit Delete wäre wir auf der sicheren Seite. Ich find aber beides praktikabel. 0 wäre halt in der Theorie schon auch ein valides Limit. Preise könnten ja auch negativ werden. Auch wenn das in der Praxis vermutlich eher selten ist. Zumindest in den Szenarien die wir jetzt kennen.